src/literals.py

```diff
@@ -84,10 +86,10 @@ class Ports:
     internal: int
 
 
-SECURITY_PROTOCOL_PORTS: dict[AuthMechanism, Ports] = {
-    "SASL_PLAINTEXT": Ports(9092, 19092),
-    "SASL_SSL": Ports(9093, 19093),
-    "SSL": Ports(9094, 19094),
+SECURITY_PROTOCOL_PORTS: dict[AuthProtocol, dict[AuthMechanism, Ports]] = {
```

Comment:
**niptick** I would find a signature `dict[(AuthProtocol, AuthMechanism), Ports]` more appropriate, but it is not a strong point

Reply:
updated (i kinda like this better too)